Dockerfile

```diff
@@ -13,6 +13,11 @@ RUN npm install
 
 COPY . .
 
+RUN npm run compile
+COPY public.pem /build/
+COPY IP_DATA.bin /build/
```

Comment:
- Are these files in the root directory of the repo? I can't find the `public.pem` file.
- Also, please change `/build` to `./build`.